### DISCUSSION TITLE
Option for "__Prowler:Foobar__" generated tags to capture context that is currently missing from findings pushed to security hub.

### DISCUSSION THREAD

**Discussion by mlmerchant** [prowler-cloud/prowler#3871] [Ideas]
I noticed that when prowler generates it's inventory of AWS resources, it seem to only keep tags and discards the rest of the available context.  It would be nice, at that step, to optionally capture those extra details into prowler generated "pseudo tags".  I assume this could be done without major changes to the rest of the application.  One example might be, looking at a report, I want to know the VPC of a resource, but prowler discarded that information.  So I'd need to do additional calls post reporting to derive that. 

If you look at security hub derived findings, those details exist within the ndjson formtted results.  Prowler, on the other hand, provides very minimal if any context.

**Comment by mlmerchant** [comment]
https://github.com/prowler-cloud/prowler/blob/35c8ea5e3f5a21a7a290ad414838e0cb762fad87/prowler/providers/aws/services/vpc/vpc_service.py#L358

For the purpose of missing the VpcId, I was able to accomplish what I needed by adding logic to the above module just before the object was built for a specific subnet.  I created a "_VpcId_" tag with the value subnet["SubnetId"] and this data followed through to the finding that was generated and pushed to security hub.

So I can solve my own problem by making those adjustments manually for now.

**Reply by jfagoagas** [reply]
Hi @mlmerchant currently we are working to include context in each Prowler finding. As you pointed out right now we only collect tags but no the relationship between resources, but that's just the plan to be able to link resources in between and have findings with more than one resource associated. Would that fix it?
